pkg/sql/tpp/tpp.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 The SQLFlow Authors. All rights reserved.
```

Comment:
tpp => third_party_parser or native_parser

Reply:
go lint forbids using an underscore in the package name. And the package name is better to be consistent with the directory name. So I am keeping the `tpp`.

```
hookid: go-lint

pkg/sql/third_party_parser/third_party_parser.go:14:1: don't use an underscore in package name
Found 1 lint suggestions; failing.
pkg/sql/third_party_parser/tidb_parser.go:14:1: don't use an underscore in package name
Found 1 lint suggestions; failing.
pkg/sql/third_party_parser/tidb_parser_test.go:14:1: don't use an underscore in package name
Found 1 lint suggestions; failing.
```